src/client/components/Post.jsx

```diff
@@ -0,0 +1,51 @@
+import "../styling/post.css"
+
+const Post = () => {
+    return (
+        <main className="postPage">
```

Comment:
very clean and readable layout, descriptive classNames :)

Reply:
If we have time, we can change the camelCase to the dashes and stuff

Reply:
Not a problem with me I like CC.